Add trial details. (#3828)
platformsh/platformsh-docs: sites/friday/src/glossary/_index.md

It provides the cryptographic "S" in HTTPS.
It's often still referred to as SSL
even though it has mostly replaced SSL for online encrypted connections.

## Trial

> When you create an account on Upsun, you are also beginning a trial for your organization.

Reviewer: ```suggestion
When you create your first organization on Upsun, you are also activating your trial for that organization.
```

> When you create an organization for the first time on Upsun, you will have time and resources available so you can try out the Upsun platform.
> It has come restrictions that are relevant to your time experimenting with the platform:

Reviewer 2: ```suggestion
When you create an account on Upsun, you are also beginning a trial for your organization.
This trial grants you a certain amount of time and resources to try out the Upsun platform, and comes with the following restrictions:
```

1. A trial lasts for **5 days**, which starts automatically from creating your first organization.
1. A trial allows for **one (1)** active [project](#project) at a time.
1. For a trial project, the trial allows for **two (2)** [active environments](#active-environment).
1. At the organization level, there is a limit on the amount of [resources available to your project](/manage-resources):

    - 4.5 CPU
    - 12 GB RAM
    - 20 GB storage

> If for some reason your trial ends before adding payment details, there are some additional things to keep in mind:

Reviewer 2: ```suggestion
If your trial ends before adding payment details, there are additional things to keep in mind:
```

- When a trial expires, both the organization and the project are suspended.

> - Projects where no code has been pushed to them are deleted **1 day** after a trial expires. 
> - Projects where code _has_ been pushed to them are deleted **5 days** after the trial expires.

Reviewer 2: ```suggestion
- Projects where no code has been pushed are deleted **1 day** after a trial expires. 
- Projects where code _has_ been pushed are deleted **5 days** after the trial expires.
```
